packages/amplify-category-api/src/provider-utils/awscloudformation/container-artifacts-graphql.js

```diff
@@ -0,0 +1,132 @@
+export const containerFiles = {
+    Dockerfile: `FROM node:alpine
+
+WORKDIR /usr/src/app
+COPY package*.json ./
+RUN npm install
+COPY . .
+
+EXPOSE 3000
+
+CMD ["node", "index.js"]
+    `,
+    'index.js': `'use strict';
+
+const PORT = 3000;
+
+var express = require('express');
+var { graphqlHTTP } = require('express-graphql');
+var { buildSchema } = require('graphql');
+var Query = require('./schema.js');
+var root = require('./resolvers.js');
+
+var app = express();
+
+// Enable CORS for all methods
+app.use(function (req, res, next) {
```

Comment:
Do we need to add the below for CORS as well or are we good without adding the method header?

```
res.header("Access-Control-Allow-Methods", "GET, PUT, POST, DELETE, OPTIONS");
res.header(
        "Access-Control-Allow-Headers",
        "Content-Type,X-Amz-Date,Authorization,X-Api-Key,X-Amz-Security-Token,X-Amz-User-Agent",
    );
```

Reply:
The ones below seemed to work but now that you mention it do we need the others for tracking usage and API calls?